src/makeRobot.js

```diff
@@ -37,8 +37,63 @@
  *
  * @return {Robot}
  */
-function makeRobot(name, wheels, version) {
-  // write code here
-}
+ function makeRobot(name, wheels, version) {
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
```

Comment:
You should get name / version / wheels not from function's parameters, but from object. Use 'this.' (Use concatenation to avoid linter error for long string).
```suggestion
    get info() {
      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
    },
```